### PR TITLE
#159004381 Notify members when user joins category.

### PR DESCRIPTION
**Pull request by arthuroe**
#### What Does This PR Do?
- Adds functionality that enables members in a category recieve notification when new members join a group.
#### Description Of Task To Be Completed
- Modify notify user slack function to send direct messages
- Add job to notify members when new users joins daily
#### Any Background Context You Want To Provide?
- Members in a category have no means of knowing who joins the categories they are part of. This implementation enables the existimbers get notified of members who join a category each day.
#### How can this be manually tested?
- After the necessary user, category and interest tables have been populated appropriately.
- Run `pip install -r server/requirements.txt`
- Run `python server/manage.py crontab add`.
- Run `python server/manage.py crontab show`.
- Run `cron -e` to verify the job is created.
#### What are the relevant PT stories?
- #159004381
#### Screenshot(s)
screen shot 2018-08-06 at 11 20 02

### CODE REVIEW

**Comment by tosmak16** [comment]
Well done! You really did an awesome job implementing this task with a job scheduler @arthuroe But I feel looking at the scope of this task (based on the description of task on PT ), sending a notification once when a new user joins a group is good. Better still... I would like to know your point of view.

**Comment by arthuroe** [comment]
@tosmak16 That was the initial plan but considering a large number of users keep joining per day, it becomes hard keeping track of the different notifications. If for instance, I belong to five categories, the notifications may become a little too much.  Keep in mind the user also gets notifications for the different events created within the category which are not periodic.

File: server/graphql_schemas/utils/cron_jobs.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                created_at__gte=datetime.date.today()
+        )
+        new_users = ' '.join(
+            ['<@' + str(get_slack_user_id(user.follower.user.email)) +
+             '>' for user in new_users_in_category if not None]
+        )
+        if new_users:
+            message = new_users + ' joined ' + str(category.name) + ' today.'
+            [notify_user(message, user.follower.user.email)
+             for user in users_in_category]
```
Comment (OsayamenOsaretin) [inline server/graphql_schemas/utils/cron_jobs.py]: This is brilliant code IMHO.

File: server/a_socials/settings/base.py [hunk 2]
```diff
@@ -40,58 +40,66 @@
...
+    'django_filters',
+    'corsheaders',
+    'graphene_django',
+    'graphql_schemas',
+    'django_crontab',
 ]
 
+CRONJOBS = [
+    ('* * * * *', 'graphql_schemas.utils.cron_jobs.check_and_notify_members',
+     '>> ~/Andela-Socials/file.log')
```
Comment (OsayamenOsaretin) [inline server/a_socials/settings/base.py]: Do we need a `file.log` file?

**Comment by tosmak16** [comment]
well done @arthuroe  But what do you think if we have the `python manage.py crontab add` command in our `makefile` and `start script`?

File: server/a_socials/settings/base.py (hunk 2 above)
Comment (arthuroe) [reply server/a_socials/settings/base.py]: @OsayamenOsaretin I don't think so this was just for debug purposes, I will remove it.

**Comment by arthuroe** [comment]
Makes sense @tosmak16 , will add it.

**Pull request merged**